.github/workflows/docker_dev.yml

```diff
@@ -95,11 +119,21 @@ jobs:
           tags: |
             type=ref,event=pr
             tpye=raw,value=dev,priority=1
+
       - name: Set up QEMU
         uses: docker/setup-qemu-action@v2
+
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
+
+      - name: Check perms
+        uses: actions-cool/check-user-permission@main
+        id: checkUser
+        with:
+          require: 'write'
+
       - name: Login to GHCR
+        if: steps.checkUser.outputs.require-result == 'true'
```

Comment:
That makes it, so it won't push if user can't ?

Is there a way I (ajnart) can manually trigger the PR to Docker Image with the Dispatch option ?